src/site/content/en/blog/speed-to-business/index.md

```diff
@@ -0,0 +1,177 @@
+
+Even though the case for speed is clear, many companies still struggle with prioritizing work that will improve their site speed since they do not know exactly how it affects **their** users and as a result **their** business. 
+
+In the absence of data, it is easy to delay site speed work and focus on other tasks. A common scenario is having some people in the company recognize the importance of site speed and yet not be able to build a case for it and convince multiple stakeholders to invest accordingly. 
+
+This article provides high-level guidance on how to leverage A/B testing to evaluate the impact of site speed on business metrics therefore enabling more effective decision-making on the matter. 
+
+
+## Step 1: Pick a page to A/B test
+
+Your aim is to test the hypothesis that page speed relates to your business metrics. For simplicity's sake, you can initially limit yourself to identifying a single page for analysis. Future work can expand to multiple pages of the same type to verify findings, and then to other areas of the site. Some suggestions for where to start are provided at the bottom of this step. Several requirements drive the page selection process:
+
+
+
+*   The A/B test should only be run on the devices of mobile users. Globally, the partner sites we assist see an average of [more than 50%](https://gs.statcounter.com/platform-market-share/desktop-mobile-tablet/worldwide) (and growing!) of their traffic coming from mobile, but this can increase significantly based on [region](https://gs.statcounter.com/platform-market-share/desktop-mobile-tablet/india) and [vertical](https://www.salecycle.com/blog/stats/2019-ecommerce-year-in-review-infographic/). Mobile devices are more sensitive to slower websites due to processing and memory constraints and less stable networks. Also, on-the-go usage patterns mean expectations for speed are higher.
+*   The page you choose for testing should be an important part of your conversion funnel. 
+
+    Every site has a different purpose, so every site tracks different success metrics. These metrics are usually related to a user journey which is analyzed using a funnel. For example, users on an e-commerce website will have to navigate through a home page, category pages, product pages, and a checkout page to complete a purchase. If you're optimizing for conversions, one of those pages would be a good candidate. 
+
+*   The page should have a singular purpose. Unless your site has a very specific mission, it's usually best to avoid the temptation to select the homepage for your test. Many commercial sites' homepages are portals to a wide variety of functionality that will make your analysis noisy. For example, if you're optimizing for pageviews per session on a news site, it might be best to exclude the non-commercial portions of the site and focus on monetized sections and articles.
```

Comment:
```suggestion
*   The page should have a singular purpose. Unless your site has a very specific mission, it's usually best to avoid using the homepage for your test. Many commercial sites' homepages are portals to a wide variety of functionality that will make your analysis noisy. For example, if you're optimizing for pageviews per session on a news site, it might be best to exclude the non-commercial portions of the site and focus on monetized sections and articles.
```